packages/ethereum/contracts/src/static/stake/HoprStakeSeason8.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+import './HoprStakeBase.sol';
+
+/**
+ * Staking program - Season 8
+ * Inherit a HoprStakeBase.
+ * Include new function for owner to stake for a group of accounts
+ */
+contract HoprStakeSeason8 is HoprStakeBase {
+  using SafeERC20 for IERC20;
+
+  /**
+   * Staking season 8 starts at 1690372800 2pm CET 26th July 2023 and ends at 1694260800 2pm CET 9th Sep 2023
+   * Basic APY is 1.25 % (= 1.25 / 100 * 1e12 / (365 * 24 * 60 * 60))
+   * the boost cap is 250k xHOPR (25e22)
+  * It blocks HODLr, DAO_v2, Surveyor, Wildhorn_v2, PuzzleHunt_v1, PuzzleHunt_v2, PuzzleHunt_v3, ETH_Denver, 
+   * Lucky NFTs, PuzzleHunt_v3, Matterhorn, DAO_v3, AMA_zing, Restaker, QV_game, Meme_master, Tokenlon_AMA, 
+   * TokenlonAMA, Meme_master_v2 and Metadata_games at start
+   * @param _newOwner address Address of the new owner. In production, it's 0xD9a00176Cf49dFB9cA3Ef61805a2850F45Cb1D05
+   * @param _nftAddress address Address of the NFT contract. In production, it's 0x43d13D7B83607F14335cF2cB75E87dA369D056c7
+   * @param _lockToken address Address of the stake token xHOPR. In production, it's 0xD057604A14982FE8D88c5fC25Aac3267eA142a08
+   * @param _rewardToken address Address of the reward token wxHOPR. In production, it's 0xD4fdec44DB9D44B8f2b6d529620f9C0C7066A2c1
+
+   */
+  constructor(
+    address _newOwner,
+    address _nftAddress,
+    address _lockToken,
+    address _rewardToken
+  ) HoprStakeBase(_newOwner, 1690372800, 1694260800, 396, 25e22, _nftAddress, _lockToken, _rewardToken) {
```

Comment:
wouldnt it be better to defien the parameters and then pass them to the function? I think it would increase readabuility. I.e. start_date = 1690372800, end_date = 1694260800, base_rate = 396, boost_cap = 25e22. 

Then 

HoprStakeBase(_newOwner, start_date, end_date, base_rate, boost_cap, _nftAddress, _lockToken, _rewardToken)

Reply:
it's not possible for constructor variables in Solidity